FiveCalls/FiveCalls/ScheduleReminders.swift

```diff
@@ -0,0 +1,285 @@
+//
+//  ScheduleReminders.swift
+//  FiveCalls
+//
+//  Created by Christopher Selin on 9/8/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import SwiftUI
+
+let USE_SHEET = true
+
+struct ScheduleReminders: View {
+    @AppStorage("reminderEnabled") var remindersEnabled = false
+    @Environment(\.dismiss) var dismiss
+
+    @State var existingSelectedTime = Date.distantPast
+    @State var selectedTime = Date.distantPast
+    @State var existingSelectedDayIndices = [Int]()
+    @State var selectedDayIndices = [Int]()
+    @State var shouldShake = false
+    
+    var body: some View {
+        if !USE_SHEET {
+            ZStack {
+                DayAndTimePickers(remindersEnabled: $remindersEnabled,
+                                  selectedTime: $selectedTime,
+                                  selectedDayIndices: $selectedDayIndices,
+                                  shouldShake: $shouldShake)
+                RemindersDisabledView(remindersEnabled: $remindersEnabled)
+                Spacer()
+            }
+            .navigationTitle(R.string.localizable.scheduledRemindersTitle())
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationBarBackButtonHidden(true)
+            .toolbar {
+                ToolbarItem(placement: .navigationBarLeading) {
+                    Button(action: {
+                        onDismiss()
+                    }) {
+                        Text(R.string.localizable.doneButtonTitle())
+                            .bold()
+                    }
+                }
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Toggle(isOn: $remindersEnabled,
+                           label: {
+                        Text("")
+                    }).toggleStyle(.switch)
+                }
+            }
+            .animation(.easeInOut, value: remindersEnabled)
+            .task {
+                let notificationRequests = await UNUserNotificationCenter.current().pendingNotificationRequests()
+                let indices = indices(from: notificationRequests)
+                existingSelectedDayIndices = indices
+                selectedDayIndices = indices
+                if let trigger = notificationRequests.first?.trigger as? UNCalendarNotificationTrigger, let triggerDate = trigger.nextTriggerDate() {
+                    existingSelectedTime = triggerDate
+                    selectedTime = triggerDate
+                }
+            }
+            .onChange(of: remindersEnabled) { newValue in
+                if newValue {
```

Comment:
TODO: integrate changes made [here](https://github.com/5calls/ios/pull/336) if we merge.